datagateway_api/src/datagateway_api/icat/filters.py

```diff
@@ -51,6 +51,9 @@ def create_filter(self):
             where_filter = self.create_condition(self.field, "=", self.value)
         elif self.operation == "ne":
             where_filter = self.create_condition(self.field, "!=", self.value)
+        elif self.operation == "neq":
+            self.operation = "ne"
+            where_filter = self.create_filter()
```

Comment:
I know this is how I suggested to implement, but your code in `src/common/filters.py` has made me think of a different, perhaps better way. So checking a list of operations, rather than just one, changing `self.operation` and calling the same function again:

```suggestion
        elif self.operation in ["ne", "neq"]:
            where_filter = self.create_condition(self.field, "!=", self.value)
```